### ISSUE TITLE
cli for node with suppressed EPIPE

### ISSUE THREAD

**Issue by m59peacemaker** [mhart/epipebomb#5]
It would be awesome to have a command that is a drop-in replacement for `node`, but suppresses EPIPE so it behaves like a normal command, and command that runs a node CLI with the same behavior.

```sh
  epb-node app.js | head
  epipebomb "tape **/*.js" | head
```

 I gave this a go, but couldn't get it. Do you think it should be possible?

**Comment by mhart** [comment]
Definitely should be possible – have you tried running `node -r epipebomb <args>`? If that works you could just wrap it in a shell script or similar

**Comment by mhart** [comment]
Actually, just requiring isn't going to be enough – you'll need to create your own module that invokes the `epipebomb` function, ie, just a file with:

```js
require('epipebomb')()
```

And then use that file/module as the `-r` argument

**Comment by m59peacemaker** [comment]
I was thinking of going more extreme than that, but actually for my use case, it works. I've seen other projects that include a `register.js` file for such purposes. Here's a PR https://github.com/mhart/epipebomb/pull/6.

**Comment by m59peacemaker** [comment]
But it's still appealing, the idea that if I use a node based cli and it throws the error, and doesn't support `-r`, then I could `epipebomb "some-command"` and it will be fixed. If you like the idea, it's a touch ugly, but the only way I can think to do it is to have `epipbomb` spawn the "some-command" child process and sift through the child process output for the error and not forward it (also with a check that it is indeed the final output of the child process). I'd be glad to submit a PR.

**Comment by m59peacemaker** [comment]
It was actually much easier than all that. https://github.com/mhart/epipebomb/pull/7

**Comment by m59peacemaker** [comment]
@mhart Do you like this, or should I make a new package `epipebomb-cli` ?

**Comment by mhart** [comment]
Hey, sorry for the radio silence – I like the idea of having `register` in there – and would totally merge a PR with that and an update to the README, but I'm not too keen on the spawn-args dependency for the other CLI option because it's unnecessary for most users of this module who just want it as a library.

So in that case, maybe better to create an `epipebomb-cli` if you need that?

That said, I'd be happy including a CLI that's basically:

```bash
#!/bin/sh
node -r ./register "$@"
```

That's not quite as comprehensive as the "run any command" case that you cover – but all node commands can be run explicitly anyway (ie, `epipebomb ./node_modules/.bin/tape <tape args>`)

**Comment by m59peacemaker** [comment]
Ah that's a lot smarter. =D

**Comment by mhart** [comment]
Fixed in #8
